ui/src/views/repositories/modals/remove-import-modal/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { useMutation } from '@apollo/client'
+import { Button, Modal, Toolbar } from '@mergestat/blocks'
+import { TrashIcon, XIcon } from '@mergestat/icons'
+import React, { useCallback } from 'react'
+import { REMOVE_IMPORT } from 'src/api-logic/graphql/mutations/repos'
+import { useRepositoriesContext, useRepositoriesSetState } from 'src/state/contexts'
+import { showSuccessAlert } from 'src/utils/alerts'
+import { TEST_IDS } from 'src/utils/constants'
+import useRepoImports from 'src/views/hooks/useRepoImports'
+
+export const RemoveImportModal: React.FC = () => {
+  const [{ importToRemove }] = useRepositoriesContext()
+  const { setShowRemoveImportModal } = useRepositoriesSetState()
+  const { refetch } = useRepoImports()
+
+  const close = useCallback(() => {
+    setShowRemoveImportModal(false)
+  }, [setShowRemoveImportModal])
+
+  const [removeImport] = useMutation(REMOVE_IMPORT, {
+    onCompleted: () => {
+      showSuccessAlert('Import removed successfully')
+      refetch()
+      close()
+    }
+  })
+
+  return (
+    <Modal open onClose={close} size="sm">
+      <Modal.Header>
+        <Toolbar className="h-16 px-6">
+          <Toolbar.Left>
+            <Toolbar.Item>
+              <Modal.Title>Remove Auto Import</Modal.Title>
+            </Toolbar.Item>
+          </Toolbar.Left>
+          <Toolbar.Right>
+            <Toolbar.Item>
+              <Button
+                skin="borderless-muted"
+                onClick={close}
+                startIcon={<XIcon className="t-icon" />}
+              />
+            </Toolbar.Item>
+          </Toolbar.Right>
+        </Toolbar>
+      </Modal.Header>
+      <Modal.Body>
+        <div className='px-6 py-6'>
+          Are you sure you want to remove <b>{importToRemove?.name}</b>?
```

Comment:
```suggestion
          Are you sure you want to remove the GitHub repo auto import for the <b>{importToRemove?.name}</b> {isOrgCheck ? 'organization' : 'user'}?
```

(`isOrgCheck` is made up, would need to be filled in properly)